src/main.ts

```diff
@@ -29,7 +29,7 @@ async function main() {
     tabs = tabsToData(data);
   }
 
-  const fullTabs = layoutGantt(tabs);
+  const fullTabs = improvedLayout(tabs);
```

Comment:
Do we still need `layoutGantt` imported here? It does look like we still call it from `index.ts` and `byOwner.ts`. Should we move those to use the improved one eventually?

Reply:
Fixed